tasks/distribution/Windows.yml

```diff
@@ -12,7 +12,7 @@
     account_disabled: "{{ item.account_disabled | default('false') }}"
     groups_action: "{{ item.groups_action | default('replace') }}"
     account_locked: "{{ item.account_locked | default('false') }}"
-    password_never_expires: '{{ item.password_never_expires | default(omit) }}'
+    password_never_expires: "{{ item.password_never_expires | default('true') }}"
     update_password: "{{ item.update_password | default('on_create') }}"
     password_expired: "{{ item.password_expired | default('false') }}"
     user_cannot_change_password: '{{ item.user_cannot_change_password | default(omit) }}'
```

Comment:
Shouldn't it be false instead of omit?

Reply:
Now default is moderately true, if we set it to false the password will expire.